scripts/install.sh

```diff
@@ -162,10 +163,59 @@ _install_macos_prompt_to_update_path() {
   echo ''
 }
 
-_install_macos_extract_files() {
+#
+# Linux
+#
+
+_linux_extract_files() {
+  TMP_DIR=$1
+  TMP_FILE=$2
+
+  cd "${TMP_DIR}"
+  if ! [ -d "${BIN_DIR}" ];then 
+    echo "Creating ${BIN_DIR}"
+    mkdir -p "${BIN_DIR}"
+  fi
+
+  tar xvf "${TMP_FILE}"
+  echo 'Extracted tarball in-place'
+
+  install -t "${BIN_DIR}/" "${TMP_DIR}/imandrax-cli"
```

Comment:
It might be a bit more future proof if we just installed all of the binaries in the package instead of explicitly listing these 3.